_pages/en/GCDevelopers-Exchange.md

```diff
@@ -38,7 +38,7 @@ Since GCDevEx was last online, the BCDevExchange has had many changes (the GCDev
 Questions we are asking:
 Does the GCDevEx site meet our current requirements?
 Given that the BC Developers Exchange repo is no longer being maintained should GCDevEx still be based on this code?
-While working through these questions we stood up a working [dev version of the BC Developers Exchange for testing](https://dev.devex.smellems.com/).
```

Comment:
We say we stood up a working dev version but it's no longer there, do we need to edit this sentence?

Reply:
The heading is **Thinking about the technical aspects**.  I think it's ok to say we did it and not provide the link.  Its also no longer what we are aligning with (reusing BCdevex).